sensor/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/calibrators/utils.py

```diff
@@ -161,11 +163,11 @@ def plot_detection_set(i: int, name: str, detections: List[BoardDetection]):
         )
         plt.colorbar(rotation_heatmap_ax, ax=axes[1, i])
 
+        # cSpell:enableCompoundWords
```

Comment:
Is this for `edgecolor`? We can add it to the dictionary.

Reply:
Added it to the respective PR